src/errors.rs

```diff
@@ -0,0 +1,24 @@
+/// Error types.
+
+use bitcoin::FeeRate;
+use bitcoin::Weight;
+use std::{error::Error, fmt};
+
+#[derive(Debug)]
+pub enum LibError {
+    Multiplication(Weight, FeeRate),
+}
+
+#[cfg(any(test, feature = "rand"))]
+impl Error for LibError {}
+
+#[cfg(any(test, feature = "rand"))]
```

Comment:
I think I did that to avoid compiler warning that it's not otherwise used if the `rand` flag is not used.  If there are more selection algorithms such as BnB than this feature flag can be removed and there will no longer be compiler warnings.

Reply:
I don't get any more warning if I remove these `cfg` flags. Though there is an unused warning on the unused `FeeRate` in the `lib.rs` file if compiling without `rand` feature.

Reply:
oops seems you are right.  fixed.